app/react/Library/components/TemplatesFilter.tsx

```diff
@@ -33,13 +34,26 @@ const filterValidSelectedTemplates = (configuredFilters: string[], selectedTempl
     ? selectedTemplates.filter(t => configuredFilters.includes(t))
     : selectedTemplates;
 
+const flattenConfiguredFillters = (configuredFilters: SettingsFilterSchema[]) =>
+  configuredFilters.reduce((result: string[], filter) => {
+    if (filter.items && filter.items.length) {
+      const items = filter.items.map(item => item.id!);
+      // eslint-disable-next-line no-param-reassign
```

Comment:
this disable can be avoided if instead of concatenating just push the destructured elements with something like result.push(...items);

Reply:
can you add or update a test to cover this?